kube/services/requestor/requestor-service.yaml

```diff
@@ -0,0 +1,24 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: requestor-service
+  annotations:
+    getambassador.io/config: | # TODO do i need this?
```

Comment:
@diw2

Reply:
no - you can remove this - we started porting reverse proxy to ambassador, but never finished